### PR TITLE
[DEV-2536] Adding Unit tests for helper methods that are (1) not REST requests & (2) exported

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**High level description:**

Adds unit tests for every exported helper function. Added a test file for each helper file for read-ability and to acknowledge possible additional tests.

**Technical details:**
Trying to test the scenario for each distinct return value.

**Link to JIRA Ticket:**
[DEV-2536](https://federal-spending-transparency.atlassian.net/browse/DEV-2536)

The following are ALL required for the PR to be merged:
- [x] Frontend review completed
- [x] Unit tests added (if applicable)
- [x] Passes linter

### CODE REVIEW

File: tests/helpers/generateFilesHelper-test.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// No tests needed ATM; all API Requests
```
Comment (ebdabbs) [inline tests/helpers/generateFilesHelper-test.js]: `generateFile` has some logic to create the `callBody` that could be tested, but ultimately we should move that logic out of the helper function.

File: tests/helpers/helpHelper-test.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// No tests needed ATM; all API Requests
+// parseMarkdown, unescapeInlineHtml?
```
Comment (ebdabbs) [inline tests/helpers/helpHelper-test.js]: I think these two functions should be tested - they are different from most other helpers and might require a different format.

File: tests/helpers/reviewHelper-test.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import * as reviewHelper from "helpers/reviewHelper";
+
+describe("reviewHelper", () => {
+    describe("getFileStates", () => {
```
Comment (ebdabbs) [inline tests/helpers/reviewHelper-test.js]: Could we also test `determineExpectedPairs`?

File: tests/helpers/submissionListHelper-test.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+// No tests needed ATM; all API Requests
+// parseRecentActivity?
```
Comment (ebdabbs) [inline tests/helpers/submissionListHelper-test.js]: There was a bug that came up recently related to file size calculations (some of which happen in `parseRecentActivity`), so it should have unit tests. Or we move a generic file size calculation to it's own helper function.

File: tests/helpers/uploadHelper-test.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+// No tests needed ATM; all API Requests
```
Comment (ebdabbs) [inline tests/helpers/uploadHelper-test.js]: `prepareMetadata` could be tested

File: tests/helpers/generateFilesHelper-test.js (hunk 1 above)
Comment (MaxwellKendall) [reply tests/helpers/generateFilesHelper-test.js]: Would like to disregard this test at the moment per above comment and exclude from scope of this story.

File: tests/helpers/helpHelper-test.js (hunk 2 above)
Comment (MaxwellKendall) [reply tests/helpers/helpHelper-test.js]: @ebdabbs ,

I think mocking the data for these functions is a big effort. Not sure what's going on here and seems like we're using a third party library `markdown` to do some dirty work here.

These are the functions I mentioned that I tried to test but mocking the data for the parameters proved troublesome.

Comment (ebdabbs) [reply tests/helpers/helpHelper-test.js]: https://federal-spending-transparency.atlassian.net/browse/DEV-2918

**Pull request merged**